### DISCUSSION TITLE
Possibility to hide clef symbol

### DISCUSSION THREAD

**Discussion by mauxion** [opensheetmusicdisplay/opensheetmusicdisplay#1235] [Ideas]
Is it possible to hide clef symbol with OSMDOptions? 
If no, I beleave it could be usefull feature for all users, who use OSMD for displaing one instruments where clef sign is odd and just takes up extra space.

**Comment by sschmidTU** [comment]
You should be able to hide the clefs with `osmd.EngravingRules.RenderClefsAtBeginningOfStaffline`.
(unless you have clefs in-measure clefs in the middle of the sheet, but that shouldn't happen in your case)

**Reply by mauxion** [reply]
thank you! It works for me great!
